### PR TITLE
Feature/button functionality

### PR DESCRIPTION
**Pull request by MTCawthray**
# Thank you for pushing your code!
## What (if any) features are you implementing?
+ This updated our save button functionality to be able to leverage the data model (and local storage data) to update the DOM. We created a getIdeasFromLocalStorage() method within our Idea class that retrieves our stored JSON data, parses it, and pushes it to our global ideasList array.

## What (if anything) did you refactor?
+ We refactored the renderIdea function to use the data from the data model to update the DOM.

## Were there any issues that arose?
+ We are having a hard time triggering our renderIdea function to show our locally saved ideas on page load.

## Is there anything that you need from your teammate?
+ NA

## Any other comments, questions, or concerns?
+ We should figure out how to use a prevent default on our page load to help display our idea cards from previous sessions.

### CODE REVIEW

**Review by jphorner — APPROVED** [review]
Fantastic job on this - well done on the JSON and event bubbling!

File: index.html [hunk 1]
```diff
@@ -10,22 +10,22 @@
...
     <section class="content">
       <form class="idea-input">
         <label for="title-input" class="input-label">Title</label>
-        <input type="text" class="input-box" id="title-input"/>
+        <input type="text" class="input-box title-input"/>
         <label for="body-input" class="input-label">Body</label>
-        <input type="text" class="input-box" id="body-input"/>
-        <button class="save-input">Save</button>
+        <input type="text" class="input-box body-input"/>
+        <button disabled class="save-input dont-click">Save</button>
```
Comment (Raquelhill) [inline index.html:25]: You can remove the dont-click

**Review by Raquelhill — COMMENTED** [review]
Great job disabling the save-input button.  Moving forward, I

**Review by Raquelhill — COMMENTED** [review]
I like how you leveraged updating the data model with the local storage to then manipulate the DOM.

**Pull request merged**